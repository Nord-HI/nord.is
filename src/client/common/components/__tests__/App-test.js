import React from 'react'
import App from '../App'
import renderer from 'react-test-renderer'

it('should match its last snapshot', () => {
  const component = renderer.create(
    <App><span>hello span!</span></App>
  )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
