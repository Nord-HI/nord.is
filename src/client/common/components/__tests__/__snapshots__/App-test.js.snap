exports[`test should match its last snapshot 1`] = `
<div
  className="container">
  <span>
    hello span!
  </span>
</div>
`;
